[SMTChecker] Enable division by zero checks in CHC engine (#10078)
ethereum/solidity: Changelog.md

### 0.7.5 (unreleased)

Compiler Features:

>  * SMTChecker: Support division by zero check also by the CHC engine

Reviewer: ```suggestion
 * SMTChecker: Add division by zero checks in the CHC engine.
```
